content/en/docs/security/hosting-best-practices.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Collector hosting best practices
+linkTitle: Collector hosting
+description: Follow best practices to securely host OpenTelemetry Collector.
+weight: 15
+---
+
+When setting up hosting for OpenTelemetry (OTel) Collector, consider the
+following practices to better secure your hosting instance.
+
+## Storing configuration information securely
+
+<!--- TODO: SHOULD ensure sensitive configuration information is stored securely. How? -->
```

Comment:
Perhaps add a reference to the secrets management practices for Kubernetes?

Reply:
I added this in 73f4c38.